### PR TITLE
ci: fix mgr pdb resource count

### PR DESCRIPTION
**Pull request by parth-gr**
from recent ocs operator changes we are trying to
have more than 1 mgr by deafult,
So to imply it we need to check for the mgr pdb
which would get created if mgr count >1

### CODE REVIEW

**Comment by b-ranto** [comment]
This passed downstream testing (the deployment passed).

**Review by b-ranto — APPROVED** [review]

**Review by ebenahar — APPROVED** [review]

File: ocs_ci/ocs/cluster.py [hunk 1]
```diff
@@ -1239,10 +1239,10 @@ def validate_pdb_creation():
     )
     item_list = pdb_obj.get().get("items")
     pdb_count = constants.PDB_COUNT
-    pdb_required = [constants.MDS_PDB, constants.MON_PDB, constants.OSD_PDB]
+    pdb_required = [constants.MDS_PDB, constants.MON_PDB, constants.OSD_PDB, constants.MGR_PDB]
```
Comment (prsurve) [inline ocs_ci/ocs/cluster.py:1242]: We may need a if condition for 4.15 only

[the remark this answers is not in the document]
Comment (vavuthu) [reply ocs_ci/ocs/cluster.py:1242]: @parth-gr black check failed. could you update with above

Comment (petr-balogh) [reply ocs_ci/ocs/cluster.py:1242]: Pratik raised a valid comment that this change has to be updated with some conditions that will do this change only when OCS_VERSION is >= 4.15 to have still the master branch working with 4.14 as well.  As master will be dedicated to 4.15 only after release of 4.14, plust this branch needs to be backward compatible for N-1 version for upgrades.

Comment (petr-balogh) [reply ocs_ci/ocs/cluster.py:1242]: See here: https://github.com/red-hat-storage/ocs-ci/blob/7070c7d42a43ed94111f56f6725726c7bd1eacaa/ocs_ci/ocs/resources/mcg.py#L694 the usage of checking the version in condition.

[the remark this answers is not in the document]
Comment (parth-gr) [reply ocs_ci/ocs/cluster.py]: Updated

Comment (parth-gr) [reply ocs_ci/ocs/cluster.py]: Added

**Review by petr-balogh — APPROVED** [review]

File: ocs_ci/utility/version.py [hunk 2]
```diff
@@ -47,7 +47,7 @@ def get_semantic_version(version, only_major_minor=False, ignore_pre_release=Fal
 VERSION_4_12 = get_semantic_version("4.12", True)
 VERSION_4_13 = get_semantic_version("4.13", True)
 VERSION_4_14 = get_semantic_version("4.14", True)
-
+VERSION_4_15 = get_semantic_version("4.15", True)
```
Comment (petr-balogh) [inline ocs_ci/utility/version.py:50]: ocs_ci/utility/version.py:52:1: E302 expected 2 blank lines, found 1
  Add one more empty line below please

Comment (parth-gr) [reply ocs_ci/utility/version.py:50]: Updated

**Review by petr-balogh — APPROVED** [review]

**Pull request merged**